src/Shared/Model/PackageMetadata.cs

```diff
@@ -135,6 +135,13 @@ public class PackageMetadata
         [JsonProperty(PropertyName = "active_flag", NullValueHandling = NullValueHandling.Ignore)]
         public bool? Active { get; set; }
 
+        /// <summary>
+        /// Gets or sets if the package is verified to be a part of the prefix/scope/namespace it is a part of.
+        /// And not grandfathered in, or a 3rd party package/part of a 3rd party namespace within the namespace. 
+        /// </summary>
+        [JsonProperty(PropertyName = "verified", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
Json property name differs from NugetPackageVersionMetadata. Same concern that I don't believe this is the right terminology.